src/components/profile/DeleteAccountBottomSheet.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import { deleteAccount } from "../../apis/user";
+import { logOut } from "../../store/slices/userSlice";
+import BottomSheet from "../common/BottomSheet";
+import Button from "../common/atoms/Button";
+
+export default function DeleteAccountBottomSheet({ handler }) {
+  const [agreePolicy, setAgreePolicy] = useState(false);
+  const dispatch = useDispatch();
+
+  const handleAgreement = (e) => {
+    setAgreePolicy(e.target.checked);
```

Comment:
저희는 이미 agreePolicy라는 상태를 가지고 있기 때문에 e.target에 접근할 필요는 없고
리액트스럽게 바꾸면 아래처럼 쓸 수 있습니다!

```suggestion
    setAgreePolicy(!agreePolicy);
```

Reply:
수정하였습니다.